pkg/binding/example_implementing_test.go

```diff
@@ -46,9 +45,22 @@ func (m ExMessage) Event(b binding.EventEncoder) error {
 	return b.SetEvent(e)
 }
 
+func (m *ExMessage) IsEmpty() bool {
+	return m == nil
```

Comment:
If you get rid of  the pointer (m ExMessage) then this is (len(m)  == 0) if you keep the pointer then it should be (m == nil || len(*m) == 0)